dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,34 @@
+package io.dropwizard.auth;
+
+public class Demo {
+    public static final String RPOSITORY = "testRepo";
+    public static void main(String[] args) {
+        int wek = 4;
```

Comment:
Possible typo in `wek`, fixes:
Candidate #1 with a confidence 37%
```suggestion
        int week = 4;
```
Candidate #2 with a confidence 31%
```suggestion
        int weak = 4;
```
Candidate #3 with a confidence 31%
```suggestion
        int web = 4;
```

<!-- lookout footnote separator -->
_Comment made by the analyzer style-check._